src/utils/dbUtils.ts

```diff
@@ -30,13 +31,14 @@ export const saveToDb = (
             .join(', ')} to db: ${db}`
         )
       )
-      .catch(e =>
+      .catch(e => {
         logger(
           `Error saving document ID: ${docs
             .map(doc => doc._id)
             .join(', ')} to db: ${db}`
         )
-      )
+        slackPoster(config.slackWebhookUrl, e).catch(e)
```

Comment:
You should only post to slack if the error is "critical" (db is down or something), so maybe you should check for "trivial" errors if there are any?